docs/en/operation/Troubleshooting.md

```diff
@@ -269,6 +269,67 @@ are
 - `ADVANCED`: samples and tracks recent accesses, higher overhead
 - `PARANOID`: tracks for leaks on every resource allocation, highest overhead. 
 
+## Master Internal Monitoring
+
+Alluxio master periodically checks the master process resource usage. This interval is controlled by
+`alluxio.master.throttle.heartbeat.interval` (default 3 seconds). In the check, a point in time(PIT) resource usage is captured,
+and a certain number of PITs (`alluxio.master.throttle.observed.pit.number`, default 3) are recorded. The PIT includes the following information.
+
+```console
+directMemUsed=5268082, heapMax=59846950912, heapUsed=53165684872, cpuLoad=0.4453061982287778, pitTotalJVMPauseTimeMS=190107, totalJVMPauseTimeMS=0, rpcQueueSize=0, pitTimeMS=1665995384998}
+```
+
+- `directMemUsed`: memory allocated by ByteBuffer.allocateDirect
+- `heapMax` : the allowed max heap size
```

Comment:
Just captured from the same structure. Interesting thing is sometimes they are not same.

Reply:
if this is what i think it is, this is only a constant if xms and xmx are set to be the same, such that the jvm is preallocated its maximum. otherwise, the jvm starts with its heap at the xms value and only allocates more if it is exceeded, up until the xmx value

Reply:
I agree it should be a fixed. Sometimes the values returned are slight different. But that is good for computation and record, so just put it there.